### PR TITLE
Products Page

### LINKED ISSUE #235: Products Page
## Feature description 

Products Page UI:
image

### PR DESCRIPTION
**Pull request by rachelhox**
## Description

closes #235 

## Checklist
- [ ] Targeted PR against correct branch.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

**Comment by kwunyeung** [comment]
@rachelhox Is it R4R?

**Comment by rachelhox** [comment]
> @rachelhox Is it R4R?

Not yet! realised the desktop view is a bit off.. working on it rn

File: src/screens/products/components/desmos/index.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  return (
+    <DesmosContentCSS>
+      <div className="desktopWrapper">
+        <div className="image">
+          <img
+            src="/static/images/assets/desmos-image.png"
+            alt="Forbole Portal Screenshot"
+          ></img>
+        </div>
+        <div className="content">
```
Comment (kwunyeung) [inline src/screens/products/components/desmos/index.tsx:17]: `<li>` should be child of `<ul>` and so this should be a `<ul>` but not a `<div>`.
```suggestion
        <ul className="content">
```

File: src/screens/products/components/desmos/index.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+          <h1>{t("desmos")}</h1>
+          <li>{t("dd1")}</li>
+          <li>{t("dd2")}</li>
+          <li>{t("dd3")}</li>
+          <a href="https://www.desmos.network/">
```
Comment (kwunyeung) [inline src/screens/products/components/desmos/index.tsx:22]: If this is one of the list items, the `<a>` should be a child of `<li>`.
```suggestion
          <li><a href="https://www.desmos.network/">
```

File: src/screens/products/components/big_dipper/components/big_dipper_content/index.tsx [hunk 3]
```diff
@@ -1,23 +1,33 @@
...
+            alt="Big Dipper Background"
+            className="bdbg"
+          ></img>
           <img
             src="/static/images/assets/bigdipper-screenshot.jpg"
             alt="Big Dipper Screenshot"
+            className="bd"
           ></img>
         </div>
         <div className="content">
```
Comment (kwunyeung) [inline src/screens/products/components/big_dipper/components/big_dipper_content/index.tsx:23]: Please read the comment below.

**Review by kwunyeung — CHANGES_REQUESTED** [review]

File: src/screens/products/components/desmos/index.tsx (hunk 2 above)
Comment (rachelhox) [reply src/screens/products/components/desmos/index.tsx:22]: @kwunyeung this is the `EXPLORER -->` under the list items
image

Comment (kwunyeung) [reply src/screens/products/components/desmos/index.tsx:22]: Then the `<a>` should be a sibling of the `<ul>`.

File: src/screens/products/components/big_dipper/components/big_dipper_content/index.tsx [hunk 4]
```diff
@@ -20,15 +20,15 @@ export const BigDipperContent = () => {
             className="bd"
           ></img>
         </div>
-        <div className="content">
+        <ul className="content">
           <h1>{t("bigDipper")}</h1>
           <li>{t("bd1")}</li>
           <li>{t("bd2")}</li>
           <li>{t("bd3")}</li>
           <a href="https://cosmos.bigdipper.live/">
```
Comment (kwunyeung) [inline src/screens/products/components/big_dipper/components/big_dipper_content/index.tsx:28]: Same for this `<a>`.

**Review by kwunyeung — CHANGES_REQUESTED** [review]

**Review by kwunyeung — APPROVED** [review]

**Pull request merged**